ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -237,21 +235,36 @@ open class CommandBar: UIView {
         if !leadingCommandGroupsView.isHidden {
             let leadingOffset = max(0, scrollView.contentOffset.x)
             let percentage = min(1, leadingOffset / scrollView.contentInset.left)
-            locations[1] = CommandBar.fadeViewWidth / containerView.frame.width * percentage
+            locations[1] = LayoutConstants.fadeViewWidth / containerView.frame.width * percentage
         }
 
         if !trailingCommandGroupsView.isHidden {
             let trailingOffset = max(0, mainCommandGroupsView.frame.width - scrollView.frame.width - scrollView.contentOffset.x)
             let percentage = min(1, trailingOffset / scrollView.contentInset.right)
-            locations[2] = 1 - CommandBar.fadeViewWidth / containerView.frame.width * percentage
+            locations[2] = 1 - LayoutConstants.fadeViewWidth / containerView.frame.width * percentage
         }
 
         containerMaskLayer.locations = locations.map { NSNumber(value: Float($0)) }
     }
 
-    private static let fadeViewWidth: CGFloat = 16.0
-    private static let fixedButtonSpacing: CGFloat = 2.0
-    private static let insets = UIEdgeInsets(top: 8.0, left: 8.0, bottom: 8.0, right: 8.0)
+    /// Updates the provided `CommandBarCommandGroupsView` with the `items` array and marks the view as needing a layout
+    private func set(_ commandGroupsView: CommandBarCommandGroupsView, with items: [CommandBarItemGroup]?) {
```

Comment:
nit: The name "set" seems a bit too generic.
How about something like: 

`private func setupGroupsView(_ commandGroupsView: CommandBarCommandGroupsView, with items: [CommandBarItemGroup]?)`